.github/workflows/release.yml

```
@@ -0,0 +1,336 @@
+        
+        cd release
+        
+        # Copy configuration and service files
+        cp ../configs/config.yaml .
+        if [ "${{ matrix.goos }}" = "linux" ]; then
+          cp ../systemd/framework-led-daemon.service .
+        fi
+        
+        # Create README for the package
+        cat > README.txt << EOF
+Framework LED Matrix Daemon ${VERSION}
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 132: could not find expected ':'

(syntax-check)